src/Lykke.Sdk/Health/HealthIssue.cs

```diff
@@ -0,0 +1,20 @@
+﻿using JetBrains.Annotations;
+
+namespace Lykke.Sdk.Health
+{
+    [PublicAPI]
+    public class HealthIssue
```

Comment:
This already exist in Lykke.Common - https://github.com/LykkeCity/CommonDotNetLibraries/blob/master/src/Lykke.Common/Health/HealthIssue.cs

Reply:
Do we need to remove it from Lykke.Common?

Reply:
I think we should mark them as obsolete for a start